jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/LambdaDaemonHost.kt

```diff
@@ -96,12 +95,8 @@ class LambdaDaemonHost(project: Project) : LifetimedProjectComponent(project) {
         val configurationType = ConfigurationTypeUtil.findConfigurationType(LambdaRunConfigurationType::class.java)
         val runConfigurations = runManager.getConfigurationsList(configurationType)
 
-        val isDebug = executor is DefaultDebugExecutor
-
         var settings = runConfigurations.filterIsInstance<LocalLambdaRunConfiguration>().firstOrNull { configuration ->
-            configuration.handler() == handler &&
-                // TODO: Get rid of the check when SAM CLI provide support for dotnet 3.1 lambda debug
-                (if (isDebug) configuration.runtime() != Runtime.DOTNETCORE3_1 else true)
+            configuration.handler() == handler
```

Comment:
do we need a sam version check then?

Reply:
We don't have an easy way to add it today since when we validate run configs we don't know run vs debug...

Our only option is to all of dotnet 3.1 behind the 1.4.0 gate and I dont really want to...